git_repo/repo.py

```diff
@@ -163,15 +163,16 @@ def _guess_repo_slug(self, repository, service, resolve_targets=None):
         for remote in repository.remotes:
             if remote.name in targets:
                 for url in remote.urls:
-                    if url.startswith('https'):
-                        if url.endswith('.git'):
-                            url = url[:-4]
+                    if url.endswith('.git'):
+                        url = url[:-4]
+                    if url.startswith('https') or url.startswith('http'):
                         *_, user, name = url.split('/')
                         self.set_repo_slug('/'.join([user, name]))
                         return
+                    elif url.startswith('ssh://'):
```

Comment:
OK I will fix it, but I think it is more better:
```python
if url.startswith('https://') or url.startswith('http://') or url.startswith('ssh://'):
```

because, "An alternative scp-like syntax" can use any username such as:

- `http@server:owner/repo.git`
- `https@server:owner/repo.git`
- `ssh@server:owner/repo.git`

I know these format can be used by Gogs. Gogs can configure ssh username. (Of course it is rare case)